### PR TITLE
Fixing contact disclose attribute parsing issue

### PR DESCRIPTION
**Pull request by skalsangrah-godaddy**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,6 +15,7 @@ apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: "org.sonarqube"
 apply plugin: 'jacoco'
+apply plugin: 'eclipse'
```
Comment (bchen-godaddy) [inline build.gradle]: Could you please remove this change - there is no need to introduce it (as there is no IntelliJ plugin used here either).

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java [hunk 2]
```diff
@@ -183,7 +182,7 @@ public void fromXML(XMLDocument xmlDoc) {
             String iCC = xmlDoc.getNodeValue(CON_PINFO_INT_CC_EXPR);
 
             intPostalInfo = new IntPostalInfo(iName, iOrg, iStreet, iCity,
-                    iSP, iPC, iCC);
+                iSP, iPC, iCC);
```
Comment (bchen-godaddy) [inline src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Could you please undo this change? It's irrelevant to the fix.

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java [hunk 3]
```diff
@@ -196,7 +195,7 @@ public void fromXML(XMLDocument xmlDoc) {
                 String lCC = xmlDoc.getNodeValue(CON_PINFO_LOC_CC_EXPR);
 
                 locPostalInfo = new LocalPostalInfo(lName, lOrg, lStreet, lCity,
-                        lSP, lPC, lCC);
+                    lSP, lPC, lCC);
```
Comment (bchen-godaddy) [inline src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Same here - please revert.

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java [hunk 4]
```diff
@@ -242,45 +242,53 @@ public void fromXML(XMLDocument xmlDoc) {
...
+    @Override
     protected String statusCountExpr() {
         return CON_STATUS_COUNT_EXPR;
     }
 
-    protected static String exprReplace(String expr) {
+    protected static String exprReplace(final String expr) {
         return expr.replaceAll(OBJ,
-                StandardObjectType.CONTACT.getName());
+            StandardObjectType.CONTACT.getName());
```
Comment (bchen-godaddy) [inline src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Same here - please revert.

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java [hunk 5]
```diff
@@ -49,8 +53,8 @@ public void testGetIntPostalInfo() {
         assertEquals("John Doe", pi.getName());
         assertEquals("Example Inc.", pi.getOrganisation());
         assertArrayEquals(
-                new String[] {"123 Example Dr.", "Suite 100"},
-                pi.getStreet());
+            new String[] {"123 Example Dr.", "Suite 100"},
+            pi.getStreet());
```
Comment (bchen-godaddy) [inline src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Same here - please revert.

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java [hunk 6]
```diff
@@ -100,15 +104,15 @@ public void testGetROID() {
     @Test
     public void testGetCreateDate() {
         assertEquals(
-                EPPDateFormatter.fromXSDateTime("1999-04-03T22:00:00.0Z"),
-                response.getCreateDate());
+            EPPDateFormatter.fromXSDateTime("1999-04-03T22:00:00.0Z"),
+            response.getCreateDate());
```
Comment (bchen-godaddy) [inline src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Same here - please revert.

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java [hunk 7]
```diff
@@ -100,15 +104,15 @@ public void testGetROID() {
...
     }
 
     @Test
     public void testGetUpdateDate() {
         assertEquals(
-                EPPDateFormatter.fromXSDateTime("1999-12-03T09:00:00.0Z"),
-                response.getUpdateDate());
+            EPPDateFormatter.fromXSDateTime("1999-12-03T09:00:00.0Z"),
+            response.getUpdateDate());
```
Comment (bchen-godaddy) [inline src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Same here - please revert.

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java [hunk 8]
```diff
@@ -129,10 +133,47 @@ public void testGetSponsorClient() {
...
-                new Status[] {
-                        new Status("linked"),
-                        new Status("clientDeleteProhibited")
-                },
-                response.getStatuses());
+            new Status[] {
+                new Status("linked"),
+                new Status("clientDeleteProhibited")
+            },
+            response.getStatuses());
```
Comment (bchen-godaddy) [inline src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Same here - please revert.

**Review by bchen-godaddy — CHANGES_REQUESTED** [review]
Thank you.

The bug fix looks good, just a few comments regarding changes that are irrlevant to the fix.

File: build.gradle (hunk 1 above)
Comment (skalsangrah-godaddy) [reply build.gradle]: Removed

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java (hunk 2 above)
Comment (skalsangrah-godaddy) [reply src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Done

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java (hunk 3 above)
Comment (skalsangrah-godaddy) [reply src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Done

File: src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java (hunk 4 above)
Comment (skalsangrah-godaddy) [reply src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java]: Done as requested.

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java (hunk 5 above)
Comment (skalsangrah-godaddy) [reply src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Done

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java (hunk 6 above)
Comment (skalsangrah-godaddy) [reply src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Done

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java (hunk 7 above)
Comment (skalsangrah-godaddy) [reply src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Done

File: src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java (hunk 8 above)
Comment (skalsangrah-godaddy) [reply src/test/unit/java/godaddy/registry/jtoolkit2/se/ContactInfoResponseTest.java]: Done
